fiona/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: fiona
+  version: {{environ['CBUILD_FIONA_VERSION']}}
+
+source:
+  git_tag: {{environ['CBUILD_FIONA_TAG']}}
+  git_url: https://github.com/Toblerity/Fiona
```

Comment:
If I got you you said you are using `master` because of `gdal 2.0.0`, right?

If so consider pinning `gdal to <2.0.0` in `ocgis` instead.  We avoid as much as we can to use `master/dev` versions.